gallery/src/main.scss

```diff
@@ -25,7 +25,7 @@
 }
 
 .nx-tree-view__child.gallery-nav-link {
-  font-size: 16px;
+  font-size: 14px;
```

Comment:
The mockup I have shows the text as 16px, which suggests that it's actually `.gallery-link` that's incorrect.

Reply:
> The mockup I have shows the text as 16px, which suggests that it's actually .gallery-link that's incorrect.

8cd4e334